librz/analysis/op.c

```diff
@@ -586,45 +530,39 @@ RZ_API const char *rz_analysis_stackop_tostring(int s) {
 	return "unk";
 }
 
-RZ_API const char *rz_analysis_op_family_to_string(int n) {
-	switch (n) {
-	case RZ_ANALYSIS_OP_FAMILY_UNKNOWN: return "unk";
-	case RZ_ANALYSIS_OP_FAMILY_CPU: return "cpu";
-	case RZ_ANALYSIS_OP_FAMILY_SECURITY: return "sec";
-	case RZ_ANALYSIS_OP_FAMILY_FPU: return "fpu";
-	case RZ_ANALYSIS_OP_FAMILY_MMX: return "mmx";
-	case RZ_ANALYSIS_OP_FAMILY_SSE: return "sse";
-	case RZ_ANALYSIS_OP_FAMILY_PRIV: return "priv";
-	case RZ_ANALYSIS_OP_FAMILY_THREAD: return "thrd";
-	case RZ_ANALYSIS_OP_FAMILY_CRYPTO: return "crpt";
-	case RZ_ANALYSIS_OP_FAMILY_IO: return "io";
-	case RZ_ANALYSIS_OP_FAMILY_VIRT: return "virt";
+static const struct {
+	int id;
+	const char *name;
+} op_families[] = {
+	{ RZ_ANALYSIS_OP_FAMILY_CPU, "cpu" },
+	{ RZ_ANALYSIS_OP_FAMILY_FPU, "fpu" },
+	{ RZ_ANALYSIS_OP_FAMILY_MMX, "mmx" },
+	{ RZ_ANALYSIS_OP_FAMILY_SSE, "sse" },
+	{ RZ_ANALYSIS_OP_FAMILY_PRIV, "priv" },
+	{ RZ_ANALYSIS_OP_FAMILY_VIRT, "virt" },
+	{ RZ_ANALYSIS_OP_FAMILY_CRYPTO, "crpt" },
+	{ RZ_ANALYSIS_OP_FAMILY_IO, "io" },
+	{ RZ_ANALYSIS_OP_FAMILY_SECURITY, "sec" },
+	{ RZ_ANALYSIS_OP_FAMILY_THREAD, "thread" },
+};
+
+RZ_API const char *rz_analysis_op_family_to_string(int id) {
+	int i;
+
+	for (i = 0; i < sizeof(op_families) / sizeof(op_families[0]); i++) {
```

Comment:
```suggestion
	for (i = 0; i < RZ_ARRAY_SIZE(op_families); i++) {
```
We have a macro for this, please use it also below :)